synopsys-task/src/synopsys-task/input.ts

```diff
@@ -22,3 +22,26 @@ export const POLARIS_ASSESSMENT_TYPES = taskLib.getDelimitedInput(
 );
 export const POLARIS_SERVER_URL =
   taskLib.getInput(constants.POLARIS_SERVER_URL_KEY) || "";
+
+// Coverity related inputs
```

Comment:
Is this comments required?

Reply:
following same strategy as polaris, we have comments above